src/types/interfaces.ts

```diff
@@ -1277,5 +1283,9 @@ export interface IDataChannelBandwidthAndRates {
   rateFromClient: number;
 }
 
+export interface ICustomHeader {
```

Comment:
NIT: `ICustomHeaders` seems more correct.

Reply:
You're right it would make more sense but I'm going to leave it because it follows the same name in streaming-client and client-logger.